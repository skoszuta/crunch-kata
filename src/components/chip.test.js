import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Chip from './chip';

describe('Chip', () => {
  const exampleContent = 'Example content';

  it('should render correctly', () => {
    const wrapper = shallow(
      <Chip>{exampleContent}</Chip>
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('should render an arrow when requested', () => {
    const wrapper = shallow(
      <Chip arrow={true}>{exampleContent}</Chip>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
