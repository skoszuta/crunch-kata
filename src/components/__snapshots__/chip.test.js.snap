// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip should render an arrow when requested 1`] = `
<styled.li>
  <styled.div>
    Example content
  </styled.div>
  <styled.div />
</styled.li>
`;

exports[`Chip should render correctly 1`] = `
<styled.li>
  <styled.div>
    Example content
  </styled.div>
</styled.li>
`;
